WebRoot/css/zm.css

```diff
@@ -3807,6 +3809,37 @@ HTML>BODY .appt-selected .appt_allday_body
 	padding-right: 0;
 }
 
+/* Out-of-Office preference section */
+.ZmOutOfOfficeWrapper {
+	@PrefOOOWrapper@
```

Comment:
if it serves purpose of allowing skin developer to provide custom styles then probably we should preserve it